src/client.rs

```diff
@@ -1282,6 +1313,47 @@ impl<T: Read + Write> Session<T> {
         .and_then(|lines| parse_id_seq(&lines, &mut self.unsolicited_responses_tx))
     }
 
+    /// The [`SETQUOTA` command](https://datatracker.ietf.org/doc/html/rfc2087#section-4.1)
+    ///
+    /// Updates the resource limits for a mailbox. Any previous limits for the named quota
+    /// are discarded.
+    /// Returns the updated quota.
```

Comment:
```suggestion
    ///
    /// Returns the updated quota.
```
Note the added newline. Without the extra newline, the last line will be rendered as part of the previous paragraph.